Description: Fix missing crtbeginS.o on mipsel
 Add mipsel to the case statement, so the gcc library folder
 is defined in /etc/fpc/cfg
Forwarded: No
Author: Peter Blackman <peter@pblackman.plus.com>
Last-Update: 2023-03-14

Index: b/fpcsrc/packages/fpmkunit/src/fpmkunit.pp
===================================================================
--- a/fpcsrc/packages/fpmkunit/src/fpmkunit.pp
+++ b/fpcsrc/packages/fpmkunit/src/fpmkunit.pp
@@ -2813,6 +2813,7 @@
       powerpc:  result := GetGccDirArch('cpupowerpc','-m32');
       powerpc64:result := GetGccDirArch('cpupowerpc64','-m64');
       aarch64:  result := GetGccDirArch('cpuaarch64','');
+      mipsel:   result := GetGccDirArch('cpumipsel','');
     end {case}
   else if OS = darwin then
     case CPU of
